settings.gradle.kts

```diff
@@ -84,3 +117,8 @@ buildCache {
         }
     }
 }
+
+fun include(name: String, path: String) {
+    include(name)
+    project(name).projectDir = File(rootDir, path)
+}
```

Comment:
why is this needed?

Reply:
Good catch. It was an artifact from a previous troubleshooting effort. I removed it.